Orange/data/pandas_compat.py

```diff
@@ -76,3 +76,42 @@ def _is_datetime(s):
                             np.column_stack(X) if X else np.empty((df.shape[0], 0)),
                             None,
                             np.column_stack(M) if M else None)
+
+
+def table_to_frame(tab):
+    """
+    Convert Orange.data.Table to pandas.DataFrame
+
+    Parameters
+    ----------
+    tab : Table
+
+    Returns
+    -------
+    pandas.DataFrame
+    """
+    def _column_to_series(col, vals):
+        print(col.name)
```

Comment:
I guess this was used for debugging. Please remove.

Reply:
thanks